compiler/AST/checkAST.cpp

```diff
@@ -247,6 +247,7 @@ void checkPrimitives()
      case PRIM_CAPTURE_FN:
      case PRIM_CREATE_FN_TYPE:
      case PRIM_STRING_COPY:
+     case PRIM_MEMCPY:
```

Comment:
I'm not to worried about identifying the FnSymbol to call - the part I'm worried about is that any call I add won't be resolved... Especially since I need the memcpy to work with ref(someChapelVar)... so ideally I would be matching a memcpy with a generic ref argument.